lib/core/src/lib/form/components/widgets/number/number.widget.html

```diff
@@ -9,7 +9,7 @@
         </label>
     </div>
     <div>
-        <mat-form-field [hideRequiredMarker]="true" class="adf-form-field-input" [floatLabel]="field.placeholder ? 'always' : 'auto'">
+        <mat-form-field [hideRequiredMarker]="true" class="adf-form-field-input" [floatLabel]="getFloatLabel()">
```

Comment:
please avoid having functions called into the template as this leads to performance issue as they are called an incredible amount of times